guidelines/index.html

```diff
@@ -3609,7 +3609,7 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </section>
          		<section id="enabling-funders"><div class="header-wrapper"><h3 id="b-3-enabling-funders"><bdi class="secno">B.3 </bdi>有効な資金提供者</h3><a class="self-link" href="#enabling-funders" aria-label="Permalink for Appendix B.3"></a></div>
 	
-	<p> Federal funds from the Health and Human Services, National Institute on Disability, Independent Living, and Rehabilitation Research (NIDILRR), initially under contract number ED-OSE-10-C-0067, then under contract number HHSP23301500054C, and now under HHS75P00120P00168. この出版物の内容は、必ずしも米国保健福祉省又は米国教育省の見解や政策を反映するものではなく、また商品名、商用製品、組織の言及は米国政府による支持を意味するものではない。</p>
+	<p>この出版物は、当初は契約番号 ED-OSE-10-C-0067 のもとで、その後契約番号 HHSP23301500054C のもとで、現在は契約番号 HHS75P00120P00168 のもとで米国保健福祉省・障害者自立生活リハビリテーション研究所 (NIDILRR) の政府資金によって一部賄われている。この出版物の内容は、必ずしも米国保健福祉省又は米国教育省の見解や政策を反映するものではなく、また商品名、商用製品、組織の言及は米国政府による支持を意味するものではない。</p>
```

Comment:
ここも既存部分なのですが (今回の修正は契約番号が1つ追加されただけ)、"U.S. Federal funds" を「政府資金」と一般語に訳しているのはざっくりしすぎのように思いました。ここは謝辞の記載なので、固有名を指すものは固有名とわかるように訳出したほうが良いと思います。

U.S. Federal fundsは「米国連邦準備預金」でしょうか。[フェデラル・ファンドについて書かれた財務省の文書](https://www.mof.go.jp/public_relations/finance/202205/202205d.html) などでは「準備預金」とされているようです。「準備資金」「準備金」などでも良いかも。

このような感じでどうでしょうか?

```suggestion
	<p>この出版物は、当初は契約番号 ED-OSE-10-C-0067 のもとで、その後契約番号 HHSP23301500054C のもとで、現在は契約番号 HHS75P00120P00168 のもとで米国保健福祉省・障害者自立生活リハビリテーション研究所 (NIDILRR) の米国連邦準備預金によって一部賄われている。この出版物の内容は、必ずしも米国保健福祉省又は米国教育省の見解や政策を反映するものではなく、また商品名、商用製品、組織の言及は米国政府による支持を意味するものではない。</p>
```

Reply:
ここで言っているのは、政府の予算の話のはずで、中央銀行の話ではないはずです。

[参議院のサイトにあるアメリカの予算制度と財政規律](https://www.sangiin.go.jp/japanese/annai/chousa/keizai_prism/backnumber/h28pdf/201614903.pdf)によれば、

> 統合予算はまず、「連邦基金」（federal funds）と「信託基金」（trust funds）という２種類の基金から成っている。

とあり、「米国連邦基金」とするのが適当でしょうか…？